assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -279,144 +234,17 @@ function BannerNotification( props ) {
 
 	const closedClass =
 		! isNavigatingToCTALink && isClosed ? 'is-closed' : 'is-open';
-	const inlineLayout = 'large' === format && 'win-stats-increase' === type;
```

Comment:
I'm not entirely clear why this has been removed. Is it because `'win-stats-increase'` is no longer an applicable type?

If so, we still have a reference to it in the SCSS, so presumably this should also be removed...

https://github.com/google/site-kit-wp/blob/df1414be151340d6a988270f4ca9d75ab51eb1b1/assets/sass/components/global/_googlesitekit-publisher-wins.scss#L134